app/posts/manage-teacher-training-applications/2021-03-18-self-service-registration.md

```diff
@@ -0,0 +1,427 @@
+---
+title: Self-service registration for accrediting bodies and their training providers
+description: Let higher education institutions register on Manage and invite their school direct partners
```

Comment:
```suggestion
description: Exploring ways to help accredited body users invite users at their partner training providers to create accounts to use the Manage service
```

Reply:
We refer to "higher education institutions" rather than "accredited bodies" since the latter could also include SCITTS, which aren't part of this work. We've also used "school direct partners" in relation to training providers to be more explicit.

I will combine your suggestion with the preferred nomenclature.